android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -178,6 +180,15 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
     }
   }
 
+  @VisibleForTesting
+  internal fun getResourcesFromParamsForQR(): Map<ResourceType?, String> {
+    val resourceMap =
+      actionParams
+        .filter { it.paramType == ActionParameterType.QUESTIONNAIRE_RESPONSE_POPULATION_RESOURCE }
+        .associate { it.resourceType to it.value }
+    return resourceMap
+  }
```

Comment:
Can be converted to expression function to improve readability 
```suggestion
  internal fun getResourcesFromParamsForQR(): Map<ResourceType?, String>  =
      actionParams
        .filter { it.paramType == ActionParameterType.QUESTIONNAIRE_RESPONSE_POPULATION_RESOURCE }
        .associate { it.resourceType to it.value }
     
```

Reply:
okay